src/Entity/Event.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\EventRepository")
+ */
+class Event
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=1000)
+     */
+    private $shortDescription;
+
+    /**
+     * @ORM\Column(type="string",nullable=true)
+     */
+    private $longDescription;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $beginningDate;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $endDate;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $location;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\Country", inversedBy="events")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $country;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $priceMin;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $priceMax;
```

Comment:
@daria-sieroshtan 
To be nice to your user you should not force price to be range, it can be single number.
One way to achieve this is to make "max price" nullable and have separate logic how to display price, eg
```
// this is separate Twig file to be included everywhere
{% if event.price_max %}
{{ event.price_max}}-{{ event.price_min}}
{% else %}
{{ event.price_min}}
{% endif %}
```

Reply:
But is is already nullable=true